maintenance/migratestatus/migratestatus.go

```diff
@@ -84,11 +84,10 @@ func main() {
 
 	// Note that continueOnError is false by default so that errors can be addressed when they occur
 	// instead of blindly continueing to the next PR, possibly continuing to error.
+	dryRun := !modify
 	m := migrator.New(*mode, strings.TrimSpace(string(tokenData)), org, repo, dryRun, continueOnError)
 
-	prOptions := &github.PullRequestListOptions{
-		State: "all",
-	}
+	prOptions := &github.PullRequestListOptions{}
```

Comment:
The State field defaults to "open" if not specified. Is it alright to ignore closed PRs and leave them with the old contexts?